tests/test_exceptions.py

```diff
@@ -0,0 +1,43 @@
+from collections import namedtuple
+
+import unittest
+from response_operations_ui.exceptions.exceptions import ApiError, InternalError, UpdateContactDetailsException,\
+    SearchRespondentsException
+
+fake_response = namedtuple('Response', 'url status_code text exception')
+
```

Comment:
Only one of the exceptions in the `exceptions.py` file is mine, however I added tests for all of them (except one which doesn't return anything at all) because it was an opportunity to do this.